Mathlib/Algebra/Group/Defs.lean

```diff
@@ -104,8 +104,8 @@ class SMul (M : Type u) (α : Type v) where
   but it is intended to be used for left actions. -/
   smul : M → α → α
 
-@[inherit_doc] infixl:65 " +ᵥ " => HVAdd.hVAdd
-@[inherit_doc] infixl:65 " -ᵥ " => VSub.vsub
+@[inherit_doc] infixr:65 " +ᵥ " => HVAdd.hVAdd
+@[inherit_doc] infixr:65 " -ᵥ " => VSub.vsub
```

Comment:
I understand why you are changing `+ᵥ`, but why are you changing `-ᵥ`?

Reply:
Hmm, now that I think about it again, I agree that this seems to be mistaken.
```suggestion
@[inherit_doc] infixl:65 " -ᵥ " => VSub.vsub
```